rukundoeric/Catalogue-of-Statistics
# Catalogue of Statistics

## What does this do? 
*  Meals-Bo app. 
## Description of Task to be completed? 
*  In this project, I implemented web-app that will help users to find meals and more details about them.
## How should this be manually tested? 
NOTE: Make sure you have Node.js and npm or yarn installed in your PC.
- Clone this repo: `git clone https://github.com/rukundoeric/Catalogue-of-Statistics.git`
- Navigate into project directory: `cd catalogue-of-statistics`
- Install all packages: `yarn install or npm install`
- Start the app: `yarn start or npm start`
